llvm-remote-index.md

```diff
@@ -0,0 +1,67 @@
+# LLVM remote index service
+
+There's currently a test instance of clangd remote-index for LLVM project at
+`llvm.clangd-index-staging.dev`. The service is maintained at best effort,
+and is not an official approved by LLVM yet.
+
+After getting a clangd with remote-index support as described in
+[here](./remote-index.md#getting-remote-index-support-in-clangd), you can
+point it at `llvm.clangd-index-staging.dev:50051` either through
+[config](./config.md) or [command line
+flags](`--remote-index-address=llvm.clangd-index-staging.dev:50051` and
+`--project-root` --- absolute path pointing to the root of your
+`llvm-project` directory on your machine).
+
+## Overview
+
+The service is provided to make development of LLVM on less powerful machines
+easier and make the development process more accessible. The whole service
+implementation, including infrastructure, is open source and publicly
+available
+([clangd/index/remote](https://github.com/llvm/llvm-project/tree/master/clang-tools-extra/clangd/index/remote)
+for `clangd-index-server` and client implementation,
+[llvm-remote-index](https://github.com/clangd/llvm-remote-index) for
+indexing, data pipelines and deployment). The service is hosted on GCP by
+clangd team, with sponsorship of Google.
+
+Whole LLVM codebase (`-DLLVM_ENABLE_PROJECTS="all"`) is indexed on a daily
+basis, index snapshots are available in [clangd/llvm-remote-index
+releases](https://github.com/clangd/llvm-remote-index/releases). We only
+index the main branch of LLVM and check for the new idex version every 6
+hours on the deployment server. The server also checks for new versions of
+`clangd-index-server` which is fetched from [clangd/clangd
+snapshots](https://github.com/clangd/clangd/releases).
+
+## Privacy and security notice
+
+Remote index service is offered for free to everyone on internet, mainly
+benefiting people working on the open-source LLVM project. The service is run
+on best-effort basis and this is not an official LLVM product.
+
+The source code we serve is the open source version of LLVM: we fetch the
```

Comment:
i've found the usage of "we" in this paragraph a little bit excessive again, feel free to paraphrase. but not too important, paragprah looks good otherwise.